src/main/java/org/flockofseagles/UI/GUI.java

```diff
@@ -0,0 +1,150 @@
+package org.flockofseagles.UI;
+
+import com.sun.tools.javac.Main;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.NodeOrientation;
+import javafx.geometry.Pos;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.canvas.Canvas;
+import javafx.scene.control.MenuBar;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+public class GUI extends Application
+{
+
+	PlayField field;
+
+	@Override
+	public void start(Stage primaryStage) throws Exception
+	{
+		Parent root = FXMLLoader.load(getClass().getClassLoader().getResource("Document.fxml"));
+		field = OptionsLayoutController.field;
+		Scene scene = new Scene(root, 400, 600);
+		scene.setOnKeyPressed(this::onKeyPressed);
+		primaryStage.setTitle("Trivia Maze");
+		primaryStage.setScene(scene);
+		primaryStage.show();
+	}
+
+	public void onKeyPressed(KeyEvent keyEvent)
+	{
+		if(new KeyCodeCombination(KeyCode.UP).match(keyEvent))
+		{
+			System.out.println("pressed");
+			field.updatePlayer(1);
+		}
+		else if(new KeyCodeCombination(KeyCode.DOWN).match(keyEvent))
+		{
+
+			field.updatePlayer(2);
+		}
+		else if(new KeyCodeCombination(KeyCode.LEFT).match(keyEvent))
+		{
+			field.updatePlayer(3);
+		}
+		else if(new KeyCodeCombination(KeyCode.RIGHT).match(keyEvent))
+		{
+			field.updatePlayer(4);
+		}
+	}
+
+
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
What happened with all the white space here lolol

Reply:
Fixed in 12eca2b